web/src/components/Routes.js

```diff
@@ -46,7 +50,10 @@ export default function Routes() {
   return (
     <>
       {!load || (user != null && data === null) ? (
-        <Loading />
+        <>
+          <Loading />
+          <Logout />
```

Comment:
Why is this Logout there? This part of the code is meant to only have the `Loading` class so that whenever we are checking stuff inside the firebase (asynchronously) we see a loading page. So, we won't need the logout here.

Reply:
as written the code doesn't let you visit any of the things the router emits w/o logging out, so adding it there was necessary to use the app for me